### DISCUSSION TITLE
贴图，添加历史记录

### DISCUSSION THREAD

**Discussion by QYYE21** [Snipaste/feedback#2718] [General]
贴图，建议添加历史记录

**Comment by Geeyun-JY3** [comment]
这不是可以设置可被恢复的贴图数吗：
![image](https://github.com/Snipaste/feedback/assets/49396421/4563a674-4859-43f7-98a3-ec6ad5138a09)

如果还想要更强的贴图历史管理功能，参见：https://github.com/Snipaste/feedback/issues/2170#issuecomment-1445372616
